src/functionalTest/java/uk/gov/hmcts/reform/sendletter/controllers/CheckPrintStatusTest.java

```diff
@@ -171,7 +162,7 @@ private UUID createLetter() throws JsonProcessingException {
         DbLetter dbLetter = new DbLetter(letterId, serviceName, letter);
         Instant created = Instant.now().minus(2, ChronoUnit.DAYS);
 
-        letterRepository.save(dbLetter, created, "some-message-id");
+        letterRepository.save(dbLetter, created, MessageIdProvider.getMessageId());
```

Comment:
This all looks good to me.But can request one small change here could rename method to `MessageIdProvider.randomMessageId`.?